packages/widget/index.html

```diff
@@ -10,8 +10,15 @@
 </head>
 
 <body>
-  <sygmaprotocol-widget>
+  <sygmaprotocol-widget id="sygmaprotocolWidget" >
   </sygmaprotocol-widget>
+    <script>
+      const widget = document.getElementById('sygmaprotocolWidget');
+      widget.whitelistedSourceNetworks = ['sepolia', 'cronos'];
+      widget.whitelistedDestinationNetworks = ['mumbai', 'sepolia', 'cronos', 'rococo-phala'];
+      widget.whitelistedSourceResources = ['ERC20LRTest'];
+    </script>
```

Comment:
do we wanna keep this?

Reply:
it's just for testing the PR, I will remove before merge